src/App.tsx

```diff
@@ -1,4 +1,6 @@
-import React from 'react';
+import { useState } from 'react';
+import cn from 'classnames';
```

Comment:
For the future, better not to use such short names, yo can name it `classnames` here